libvmi/accessors.c

```diff
@@ -211,6 +211,17 @@ vmi_get_name(
     }
 }
 
+char *
+vmi_get_rekall_path(
+    vmi_instance_t vmi){
+    if (VMI_OS_LINUX == vmi_get_ostype(vmi)){
+        return (char*)((linux_instance_t)vmi->os_data)->rekall_profile;
+    }
+    else{
```

Comment:
Prefer to see a switch/case on ostype, with a default that fails cleanly (explicit return of NULL)